process/elasticproc/transactions/transactionDBBuilder.go

```diff
@@ -78,6 +78,8 @@ func (dtb *dbTransactionBuilder) prepareTransaction(
 		esdtValues = res.ESDTValues
 	}
 
+	senderUserName := converters.TruncateFieldIfExceedsMaxLength(string(tx.SndUserName))
```

Comment:
not quite needed, we have an interceptor level protection set to 32 characters. Can remain like this for extra protection.